src/api/app/controllers/webui/projects/maintained_projects_controller.rb

```diff
@@ -0,0 +1,56 @@
+module Webui
+  module Projects
+    class MaintainedProjectsController < WebuiController
+      before_action :set_project
+      before_action :set_maintained_project, except: [:index]
+      after_action :verify_authorized, except: [:index]
+
+      def index
+        respond_to do |format|
+          format.html
+          format.json do
+            render json: MaintainedProjectDatatable.new(params, view_context: view_context,
+                                                                project: @project, current_user: User.current)
+          end
+        end
+        switch_to_webui2
+      end
+
+      def destroy
+        authorize @project, :destroy?
+
+        maintenance_project = MaintainedProject.find_by!(project: @maintained_project)
+
+        @project.maintained_projects.destroy(maintenance_project)
+
+        flash_message = if @project.valid? && @project.store
+                          { success: "Removed #{maintenance_project.project} from maintenance" }
+                        else
+                          { error: "Failed to remove #{maintenance_project.project} from maintenance" }
+                        end
+
+        redirect_to(projects_project_maintained_projects_path(project_name: @project.name), flash_message)
+      end
+
+      def create
+        authorize @project, :update?
+
+        @project.maintained_projects.create!(project: @maintained_project)
+
+        flash_message = if @project.valid? && @project.store
+                          { success: "Added Maintenance to #{@maintained_project}" }
+                        else
+                          { error: "Failed to add Maintenance to #{@maintained_project}" }
```

Comment:
Let's add the validation error here: `.errors.full_messages.to_sentence`

Reply:
done!